com.microsoft.mrtk.spatialmanipulation/Tests/Runtime/ObjectManipulatorTests.cs

```diff
@@ -168,6 +169,12 @@ public IEnumerator TestObjManipSmoothingDrift()
             // Disable smoothing, to check that it properly sticks to the hand once disabled.
             objManip.SmoothingNear = false;
 
+            yield return rightHand.SetHandshape(HandshapeId.Open);
+            yield return RuntimeTestUtilities.WaitForUpdates();
+            yield return rightHand.SetHandshape(HandshapeId.Pinch);
+            yield return RuntimeTestUtilities.WaitForUpdates();
+            originalAttachOffset = attachTransform - cube.transform.position;
```

Comment:
off topic, but I've noticed recently as I was trying to write tests for #11264 that just these lines cause the cube to move even though the hand itself doesn't move. Is that what this code is trying to account for? Or are there some settings I'm missing to prevent that from happening?